daq/session_server.py

```diff
@@ -20,34 +23,44 @@
 DEFAULT_SERVER_PORT = 50051
 DEFAULT_BIND_ADDRESS = '0.0.0.0'
 DEFAULT_SERVER_ADDRESS = '127.0.0.1'
-DEFAULT_RPC_TIMEOUT_SEC = 10
+DEFAULT_RPC_TIMEOUT_SEC = 600
+
+
+# pylint: disable=no-member
```

Comment:
Seems a cleaner way is to add the proto class in .pylintrcfile..

Reply:
You are correct, thanks, I forgot about that!